client/reader/search-stream/index.jsx

```diff
@@ -165,12 +165,12 @@ class SearchStream extends React.Component {
 		return (
 			<div>
 				<DocumentHead title={ documentTitle } />
+				<ReaderBackButton />
+				<NavigationHeader
+					title={ translate( 'Search' ) }
+					subtitle={ translate( 'Search for specific topics, authors, or blogs.' ) }
+				/>
 				<div className="search-stream__fixed-area" ref={ this.handleFixedAreaMounted }>
-					<ReaderBackButton />
-					<NavigationHeader
-						title={ translate( 'Search' ) }
-						subtitle={ translate( 'Search for specific topics, authors, or blogs.' ) }
-					/>
```

Comment:
On scroll we are not keeping navigation header sticky on top so moving this out to keep it consistent with other pages.